web/src/main/resources/application.properties.EXAMPLE

```diff
@@ -1,6 +1,7 @@
 # VEP web API URL with additional parameters. Note that "VARIANT" field will be replaced with
 # the actual variant value before each VEP request.
 vep.url=http://grch37.rest.ensembl.org/vep/human/hgvs/VARIANT?content-type=application/json&xref_refseq=1&ccds=1&canonical=1&domains=1&hgvs=1&numbers=1&protein=1
+vep.max_page_size=300
```

Comment:
This is the max page size allowed by public VEP service, it might be possible to increase this for a private instance.

Reply:
maybe add comment here in the code itself